### PR TITLE
Add WOPI Business Flow

### PR DESCRIPTION
**Pull request by mmattel**
Post a discussion with @tbsbdr we more granular describe which steps are required respectively made when using WOPI and Microsoft 365. This will help readers to understand who is providing what and why and how the final license check with Microsoft is done.

When the text is agreed, we will use the same one for the ocis version.

Language review welcomed.

No merge without approval from @tbsbdr 

Backport to 10.11 and 10.10

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 1]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
 * This app requires at minimum ownCloud Version 10.5 and php 7.1.
 ====
 
+== Business Flow for Microsoft 365
+
+Apart licensing, when using Microsoft 365, the following business flow applies, contact {oc-support-url}[ownCloud Support] for more details:
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:35]:
```suggestion
Apart from licensing, when using Microsoft 365, the following business flow applies, contact {oc-support-url}[ownCloud Support] for more details:
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 2]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+
+* The customer provides ownCloud:
+** a written statement about his Microsoft 365 entitlement.
+** his ownCloud Instance URL. Only users coming from this URL will be able to use Microsoft 365.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:39]:
```suggestion
** the URL of their ownCloud instance. Only users coming from this URL will be able to use Microsoft 365.
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 3]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+* ownCloud provides the customer a required proxy URL to be used in the settings, see below.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:40]:
```suggestion
* ownCloud provides customers with a required proxy URL to be used in the settings, see below.
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 4]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+** Beside other tasks, the proxy checks if users originate from the given ownCloud Instance URL.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:41]:
```suggestion
** Among other things, the proxy checks if users originate from the given ownCloud Instance URL.
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 5]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+* When the users opens an Office document via the ownCloud Instance and Office 365 for the web is loaded, users are checked by Microsoft if already signed in with a Microsoft 365 business account. If users are not signed in, they will be prompted to sign in.
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:42]:
```suggestion
* When users open an office document via the ownCloud instance and Office 365 for the web is loaded, Microsoft checks if these users are already signed in via a Microsoft 365 business account. If users are not yet signed in, they will be prompted to sign in.
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 6]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+
 == Preparing the Environment
-You need an
 
-- {office-online-server-url}[Office Online Server] installed, or an 
-- active Microsoft 365 subscription.
+To use Microsoft Office for the web, you need an:
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:46]:
```suggestion
To use Microsoft Office for the web, you need:
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 7]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+
+- {office-online-server-url}[Office Online Server] locally installed, or an
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:48]:
```suggestion
- an {office-online-server-url}[Office Online Server] locally installed, or 
```

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 8]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
...
+
 == Preparing the Environment
-You need an
 
-- {office-online-server-url}[Office Online Server] installed, or an 
-- active Microsoft 365 subscription.
+To use Microsoft Office for the web, you need an:
+
+- {office-online-server-url}[Office Online Server] locally installed, or an 
+- active Microsoft 365 subscription including the data provided by ownCloud, see xref:business-flow-for-microsoft-365[Business Flow for Microsoft 365].
```
Comment (EParzefall) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:49]:
```suggestion
- an active Microsoft 365 subscription including the data provided by ownCloud, see xref:business-flow-for-microsoft-365[Business Flow for Microsoft 365].
```

**Review by EParzefall — APPROVED** [review]

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc [hunk 9]
```diff
@@ -30,11 +30,23 @@ Please bear in mind:
 * This app requires at minimum ownCloud Version 10.5 and php 7.1.
 ====
 
+== Business Flow for Microsoft 365
```
Comment (tbsbdr) [inline modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:33]: I would not use the wording "business flow" here as this term in this context relates to the enforced authentication via MS (`wopi.business-flow.enabled`).

**Review by tbsbdr — CHANGES_REQUESTED** [review]
"business flow" is ambiguous

File: modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc (hunk 9 above)
Comment (mmattel) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc:33]: Discussed with @tbsbdr - got the point now. Will change the headline.

Comment (mmattel) [reply modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc]: The headline is fixed now, @tbsbdr pls re-check and approve if ok.

**Review by tbsbdr — APPROVED** [review]
lgtm, thanks!

**Pull request merged**